### ISSUE TITLE
Design | Profile | Military info enhancements | Incorporate feedback from team

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/va.gov-team#72571]
## Background
We have some outstanding feedback from the team captured in Sketch comments that needs to be revisited and potentially incorporated into the design prior to design intent.

## Tasks

- [x] Review feedback from team
- [x] Update designs as needed
- [x] Add any needed notes to collab cycle ticket #72292

## AC

- [x] Design intent artifacts updated

**Comment by andaleliz** [comment]
@mtcA6 we forgot to make a ticket for this during planning last week. I've added it to the current sprint.

**Comment by andaleliz** [comment]
I reviewed the feedback from team reviews, and want to drop some comments here to acknowledge and capture what I changed, especially since @pamela-drouin is going to design intent while I'm out.  Thank you @pamela-drouin @mattmarino-adhoc @Samara-Strauss @mtcA6 @tpharrison and @adamwhitlock1 for sharing your feedback with me!

### TL;DR: 

- I made some modifications to the duty status/date section, and have updated the links in the collab cycle ticket to point the versions on [this design intent page](https://www.sketch.com/s/fc96664a-1c62-40ed-9fcd-90218c54e775/p/60B19B0B-08C8-492D-BA5B-B23AC8D76D2F/canvas) in our sketch file. 
- We also have final copy for the new FAQ we're adding to the page, and that's included on that page too.

### Details on feedback considerations
#### Whether or not the list should be an `<ol>`, `<ul>`, or `<dl>`, and if that impacts the copy/paste-ability of the text.  
- I think at this stage of the work, regardless of the markup, the bulleted list is the best presentation for scanning and understanding how content is associated to the header.  
- I experimented with copying and pasting unordered lists from VA.gov into a form (526EZ) and it pasted the content onto separate lines without bullets. I think that would work for people.

#### adding a copy/paste button. 
- I started to explore this (using the [experimental tertiary button,](https://www.sketch.com/s/fc96664a-1c62-40ed-9fcd-90218c54e775/a/DEYwRZy) and[ the secondary button](https://www.sketch.com/s/fc96664a-1c62-40ed-9fcd-90218c54e775/a/aMeaOQz)) but came up against some questions/complexities that made me feel like this wouldn't be an MVP feature and it's something better to explore later.  
- A tertiary button would probably be best for this, but we don't have guidance/components for that from the DS team yet. The current option isn't great in a vertical layout. 
- I'd also like to learn more about how veterans would want to use this data to complete their forms. 

#### Adding the total count of how many items there are to the `+5 more...` treatment. 
- I decided not to explore that for the time being since that's not an OOB option for the DS component, and 
- I think that since this is someone's personal information, they should have a sense for how many items they'll see on the list. 
- We can always revisit this if we learn it's a problem when we talk to veterans.

#### Making duty status and dates stand out more
- I explored a lot of variations with this ([available here](https://www.sketch.com/s/fc96664a-1c62-40ed-9fcd-90218c54e775/p/34A5FF7A-3D71-4DBD-93EF-022F9C1B5D4F/canvas)) and landed on a simple treatment of adding an `<hr>`, and making the duty status an h4 in option A, and reducing the heading size for details to an h5.  For option B, I opted for the `<hr>` and bolding the duty status as regular body text. It looked weird to me to have it styled as a heading with such minimal other information exposed on the page.
- I'm not sure if it makes more sense to put dates or duty first, but we can ask that of veterans when we talk to them about their data. 

I feel like we're in a great place for design intent + research, and am closing this ticket for now since most folks are out and I won't be able to change anything if design intent happens while I'm away. I think we can address further changes in a new ticket.
